### ISSUE TITLE
Permission error when viewing logs on macOS

### ISSUE THREAD

**Issue by jussapaavo** [rszyma/kanata-tray#27]
When trying to read the logs, I get `Unable to read the file`. The log files are in the correct location and can be seen with Finder, but they cannot be opened with any application. After inspecting, it seems like the log files are created without read permissions for the current user.
To fix this locally, I need to run `chown myusername /private/tmp/kanata_lastrun_3545806179.log` for each of the log files.

Is this because I run the `kanata-tray` service as `sudo`? I'm trying to research if I can fix this locally.

Here's what `ls` gives:
```sh
.rw------- root wheel 19 KB 2024-07-27 11:32:45 AM /private/tmp/kanata_lastrun_1569220938.log
```

**Comment by rszyma** [comment]
> Is this because I run the `kanata-tray` service as `sudo`

Yes. It's also a bit unfortunate that kanata need to run as superuser in macos, and there's no way around it. It's one of "security features" of macos. Log files being readable only by superuser make sense in that security framework, because if you (accidentally) enable --debug flag in kanata, you're basically keylogging yourself, but it's at least only readable to superuser.
